src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ConsoleHandler {
+    private static final String NAME_REGEX = "[a-zA-Z]+";
+    private static final String EMAIL_REGEX = "[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,6}$";
+    private static final String WRONG_DATA_INSERTION = "Будь ласка, введіть коректні дані";
+    private static final String WHITESPACE_SEPARATOR = " ";
+    private static final String QUIT_SIGH = "q";
+    private BetDao betDao = new BetDaoImpl();
+    private UserDao<User> users = new UserDaoImpl();
```

Comment:
why do we have first dao with generics and second without, let's keep our code consistent

Reply:
done